plugins/ks.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ks
+spec:
+  version: v0.0.10
+  homepage: https://github.com/linuxsuren/ks
+  shortDescription: Make it be easy to work with Kubesphere
+  description: |
+    This plugin simplfy the work when you're managing or developing with Kubesphere.
```

Comment:
Paragraph has quite a few grammatical issues. Please run it by grammarly etc. 

Also it's not a great place to define Kubesphere and how beneficial it is. You can focus on what the plugin does.

Reply:
Sorry about that. I checked my sentence with Google translation in my last commit.